test/end-to-end/pages/payment/govukconfirmpayment.js

```diff
@@ -2,10 +2,11 @@
 
 const testConfig = require('config');
 
-module.exports = function () {
+async function seeGovUkConfirmPage () {
     const I = this;
 
     I.seeInCurrentUrl(testConfig.TestGovUkConfirmPaymentUrl);
+    await I.retry(2).navByClick('#confirm');
```

Comment:
interested why we need a retry, probably need an await I.waitForElement prior to  click, I always add that in prior to clicking a button. you've also got waitForVisible and waitForClickable. more scientific and reliable 
than blunt retry